src/attributedlabel/src/NIAttributedLabel.m

```diff
@@ -214,7 +214,7 @@ @interface NIAttributedLabel() <UIActionSheetDelegate>
 
 @property (nonatomic) CTFrameRef textFrame; // CFType, manually managed lifetime, see setter.
 
-@property (assign)            BOOL detectingLinks; // Atomic.
+@property (nonatomic, assign) NSInteger linkDetectionGeneration;
```

Comment:
Yea I read through the PR more and now I understand the intent better. Would "linkDetectionRequestID" be a better name?